swift/test/Ice/interceptor/Client.swift

```diff
@@ -49,8 +49,10 @@ class InterceptorI: Disp {
                 }
             }
             current.ctx["retry"] = "no"
+        } else if current.ctx["retry"] != nil && current.ctx["retry"] == "yes" {
```

Comment:
Isn't it better to use `else if let v = current.ctx["retry"], v == "yes"` in Swift?